app/models/profile.rb

```diff
@@ -0,0 +1,59 @@
+class Profile < ActiveRecord::Base
+  include Privacy
+
+  belongs_to :user
+
+  has_attached_file :curve, styles: {
```

Comment:
> They will be profiles containing numbers in EUR 

Maybe we should call it `PriceCurve` (for consistency with [merit](https://github.com/quintel/merit/blob/00abba2c5c0ce5c627cfd00ab7451d2f8822507d/lib/merit/order.rb#L87-L94)), or `PriceProfile`? It's not really a *profile*...  a profile describes *shape* while the price curve has real values. (I'm just arguing semantics here, this isn't important :smiley:)

```
Curve
 ├ PriceCurve
 └ LoadProfile
```